src/utils/internal.ts

```diff
@@ -1809,19 +1810,19 @@ export function assertStatIsSet(
  * @param args.identity optional. If passed, Accounts that are not part of the given Identity will be filtered out
  */
 export async function getSecondaryAccountPermissions(
-  args: { accounts: Account[]; identity?: Identity },
+  args: { accounts: (Account | MultiSig)[]; identity?: Identity },
```

Comment:
The input type shouldn't need to be a union.

MultiSig extends Account so it should satisfy the smaller `Account` interface.